ci/lib/shard.sh

```diff
@@ -55,8 +55,11 @@ readonly PUBSUB_SHARD=(
 
 readonly STORAGE_SHARD=(
   storage
+  storagecontrol
```

Comment:
SGTM, but we are still missing the hand-crafted examples in `google/cloud/batch`.